makroud_test.go

```diff
@@ -3384,17 +3384,8 @@ type SetupHandler func(driver makroud.Driver)
 func Setup(t require.TestingT, options ...makroud.Option) SetupCallback {
```

Comment:
`Setup` can be private. As it's only used in `makroud_test` package. The same for all other types and functions like `Options`, `DropTables` and so on.

Reply:
I prefer to keep these functions public inside the test package as "you can use them".
Though, you make a good point about `DropTables` and `CreateTables`, they should be private.